2018-06-26 23:19:12,477 [        MainThread][         start][    INFO] Starting TheRaidMap
2018-06-26 23:19:12,477 [        MainThread][   detect_text][    INFO] Check for Loginbutton - start ...
2018-06-26 23:19:12,766 [        MainThread][   detect_text][    INFO] Check for Loginbutton - finished...
2018-06-26 23:19:12,766 [        MainThread][   detect_text][    INFO] Check for Messagebox - start ...
2018-06-26 23:19:13,059 [        MainThread][   detect_text][    INFO] Found Messagebox - closing ...
2018-06-26 23:19:13,060 [        MainThread][   detect_text][    INFO] Check for Messagebox - finished...
2018-06-26 23:19:13,060 [        MainThread][   detect_text][    INFO] Check for Xbutton - start ...
2018-06-26 23:19:13,081 [        MainThread][         start][   ERROR] Uncaught exception
Traceback (most recent call last):
  File "start.py", line 105, in <module>
    main()
  File "start.py", line 69, in main
    check_Xbutton()
  File "/Users/fameitelhuber/Desktop/TheRaidMapper/detect_text.py", line 69, in check_Xbutton
    img.save('temp.jpg')
NameError: global name 'img' is not defined
